pinc/slim_header.inc

```diff
@@ -19,7 +19,7 @@ function slim_header($title="", $extra_args=array())
     // Include the slim_header stylesheet
     if(!isset($extra_args['css_files']))
         $extra_args['css_files'] = array();
-    $extra_args['css_files'][] = "$code_url/styles/slim_header.css";
+    $extra_args['css_files'] = array_merge(["$code_url/styles/slim_header.css"], $extra_args['css_files']);
```

Comment:
I don't understand why this change was made. Is it to ensure that `slim_header.css` is always in front? If so, `array_unshift()` would be clearer.

Reply:
Yes, it is so the body margin in slim_header.css can be over-ridden. I've changed is as you suggest.